Services/Administration/OJS.Services.Administration.Business/ProblemResources/Validators/ProblemResourceDeleteValidator.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OJS.Services.Administration.Business.ProblemResources.Validators;
+
+using OJS.Services.Administration.Models.Validation;
+
+public class ProblemResourceDeleteValidator : BaseDeleteValidator<BaseDeleteValidationModel<int>, int>
+{
+}
```

Comment:
Do we need empty validator?

Reply:
It will be fixed in https://github.com/SoftUni-Internal/exam-systems-issues/issues/1180